pkg/controller/job/metrics/metrics.go

```diff
@@ -84,6 +86,18 @@ var (
 		},
 		[]string{"completion_mode", "result"})
 
+	// PodFailuresHandledByPodFailurePolicy records the number of finished Pods
+	// handled by pod failure policy.
+	// Possible label values:
+	//   action: "FailJob", "Ignore", "Count"
+	PodFailuresHandledByPodFailurePolicy = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem: JobControllerSubsystem,
+			Name:      "pod_failures_handled_by_pod_failure_policy_total",
+			Help:      "The number of failed Pods handled by pod failure policy",
```

Comment:
explain the action label and possible values

Reply:
Done